tests/ui/consts/const-unstable-intrinsic.stderr

```diff
@@ -50,7 +50,10 @@ error: `min_align_of_val` is not yet stable as a const intrinsic
 LL |         unstable_intrinsic::old_way::min_align_of_val(&x);
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
+   = note: use of unstable library feature `unstable`
```

Comment:
Why does the note repeat the error description?

The diagnostic just above doesn't do this.